### ISSUE TITLE
Remove bower dependency

### ISSUE THREAD

**Issue by epaminond** [RSSchermer/ember-spectrum-color-picker#17]
It seems bower is [getting deprecated](https://github.com/bower/bower/issues/2298) and most of the packages are using npm instead. It'd be great if ember-spectrum-color-picker also didn't have bower-dependencies.

As far as I can see it's possible [to use custom assetPath](https://github.com/RSSchermer/ember-spectrum-color-picker/commit/89816e8888b88fe74d0956dc05551afc86de450d) for that purpose but it's not released yet.

**Comment by RSSchermer** [comment]
Hey, thanks for bringing this up. I'd complete forgotten to release the custom `assetPath` PR, should now have been released as 0.5.2.

I'll also look into replacing the bower dependency entirely with an npm dependency. Do you happen to know if there's any "official" advice from the Ember team on the bower deprecation?

**Comment by epaminond** [comment]
Not yet. But on my current project `ember-spectrum-color-picker` is the last package having bower-dependency. :)

**Comment by Devoter** [comment]
Yes, it is really upsets me too. :) Please, remove bower dependence.

**Comment by mminkoff** [comment]
I wish I had time to take this on at the moment, but here's a commit from another add-on that removed a bower dependency: https://github.com/funkensturm/ember-local-storage/commit/d4c0deff28b140ccd1f410ab81a5da5e46d04391

**Comment by RSSchermer** [comment]
Alright, I've attempted a fix on branch `no-bower-dep` that seems to work with a fresh Ember 2.14.2 app. Perhaps someone here could swap out their current `ember-spectrum-color-picker` dependency with:

```
"ember-spectrum-color-picker": "rsschermer/ember-spectrum-color-picker#no-bower-dep"
```

so we can verify that it also works with a real world application.

If it indeed seems to work fine, then I'll merge into master and release as a new minor version.

**Comment by epaminond** [comment]
@RSSchermer , just tried that and it seems to work fine. :)

**Comment by RSSchermer** [comment]
Thanks for the quick response @epaminond!

I've released the fix as `0.6.0`. Closing this issue for now, but feel free to reopen if you run into problems.
